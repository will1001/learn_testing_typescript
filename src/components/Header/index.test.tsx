import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './index'

describe('About.test.js', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<About Title="test"/>)

    expect(wrapper.length).toEqual(1)

    const tree = toJson(wrapper)
    expect(tree).toMatchSnapshot()
  })
})
