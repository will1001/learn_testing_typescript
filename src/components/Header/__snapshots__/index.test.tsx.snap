// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.test.js renders as expected 1`] = `
<div>
  <h1>
    test
  </h1>
  3
</div>
`;
